examples/dt/uni01alpha/control-plane/nncp/values.yaml

```diff
@@ -0,0 +1,225 @@
+      metallb.universe.tf/loadBalancerIPs: 172.17.0.80
+    net-attach-def: |
+      {
+        "cniVersion": "0.3.1",
+        "name": "internalapi",
+        "type": "macvlan",
+        "master": "internalapi",
+        "ipam": {
+          "type": "whereabouts",
+          "range": "172.17.0.0/24",
+          "range_start": "172.17.0.30",
+          "range_end": "172.17.0.70"
+        }
+      }
+
+  storage:
+    dnsDomain: storage.openstack.lab
+    subnets:
+      - allocationRanges:
+          - end: 172.18.0.250
+            start: 172.18.0.100
+        cidr: 172.18.0.0/24
+        name: subnet1
+        vlan: 21
+    mtu: 9000
+    prefix-length: 24
+    iface: storage
+    vlan: 21
+    base_iface: enp6s0
+    lb_addresses:
+      - 172.18.0.80-172.18.0.90
+    net-attach-def: |
+      {
+        "cniVersion": "0.3.1",
+        "name": "storage",
+        "type": "macvlan",
+        "master": "storage",
+        "ipam": {
+          "type": "whereabouts",
+          "range": "172.18.0.0/24",
+          "range_start": "172.18.0.30",
+          "range_end": "172.18.0.70"
+        }
+      }
+
+  tenant:
+    dnsDomain: tenant.openstack.lab
+    subnets:
+      - allocationRanges:
+          - end: 172.19.0.250
+            start: 172.19.0.100
+        cidr: 172.19.0.0/24
+        name: subnet1
+        vlan: 22
+    mtu: 1500
+    prefix-length: 24
+    iface: tenant
+    vlan: 22
+    base_iface: enp6s0
+    lb_addresses:
+      - 172.19.0.80-172.19.0.90
+    net-attach-def: |
+      {
+        "cniVersion": "0.3.1",
+        "name": "tenant",
+        "type": "macvlan",
+        "master": "tenant",
+        "ipam": {
+          "type": "whereabouts",
+          "range": "172.19.0.0/24",
+          "range_start": "172.19.0.30",
+          "range_end": "172.19.0.70"
+        }
+      }
+
+  octavia:
+    dnsDomain: octavia.openstack.lab
+    mtu: 1500
+    vlan: 23
+    base_iface: enp6s0
+    net-attach-def: |
+      {
+        "cniVersion": "0.3.1",
+        "name": "octavia",
+        "type": "bridge",
+        "bridge": "octbr",
+        "ipam": {
+          "type": "whereabouts",
+          "range": "172.20.0.0/24",
+          "range_start": "172.20.0.30",
+          "range_end": "172.20.0.70"
+      }
+
+  external:
+    dnsDomain: external.openstack.lab
+    subnets:
+      - allocationRanges:
+          - end: 192.168.47.250
+            start: 192.168.47.1
+        cidr: 192.168.32.0/20
+        gateway: 192.168.32.1
+        name: subnet1
+    mtu: 1500
+
+  datacentre:
+    net-attach-def: |
+      {
+        "cniVersion": "0.3.1",
+        "name": "datacentre",
+        "type": "bridge",
+        "bridge": "ospbr",
+        "ipam": {}
+      }
+
+  dns-resolver:
+    config:
+      server:
+        - 192.168.122.1
+      search: []
+    options:
+      - key: server
+        values:
+          - 192.168.122.1
+
+  routes:
+    config:
+      - destination: 192.168.122.0/24
```

Comment:
with this routes added tempest tests failed for us as SSH stucks to the ctlplane ips.
without these routes it worked fine

Reply:
Thanks @karelyatin 

I have removed it and there was no impact on `controlplane` deployed or configuration.

Reply:
@psathyan why added this back? it fails again.

Reply:
Sorry... I thought I had mentioned it.

Kustomize build fails if the routes.config.0.destination is not provided.

Reply:
```
Error: accumulating components: accumulateDirectory: "recursed accumulation of path '/home/psathyan/src/architecture/dt/uni01alpha/nncp': accumulating components: accumulateDirectory: \"recursed accumulation of path '/home/psathyan/src/architecture/lib/nncp': fieldPath `data.routes.config.0.destination` is missing for replacement source ConfigMap.[noVer].[noGrp]/network-values.[noNs]\""
```

Reply:
ok for now can change it to 0.0.0.0/0 like va hci[1], with that it works fine.

[1] https://github.com/openstack-k8s-operators/architecture/blob/0ff74644ea6b85a576fe2f472b0eef691c76281b/examples/va/hci/control-plane/nncp/values.yaml#L197

Reply:
Done.